boost159/README.md

```diff
@@ -0,0 +1,15 @@
+# boost159
+
+Boost provides free peer-reviewed portable C++ source libraries.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+*TODO: Add instructions for usage*
```

Comment:
Should we do this before merging?

Reply:
@rsertelon - can you add in some brief instructions for usage?  Something like [this](https://github.com/habitat-sh/core-plans/tree/master/curl)?